client/app/scripts/superdesk/ui/ui.js

```diff
@@ -331,8 +331,23 @@ define([
                         css.right = Math.max(5, window.innerWidth - pos.left) + 'px';
 
                         dropdown.dropdownMenu.css({opacity: '0'}); // avoid flickering
-                        scope.$applyAsync(function() {
+
+                        scope.$applyAsync(function () {

```

Comment:
`$evalAsync()` is slightly better IMO, because it might execute in the same $digest cycle if one is already in progress (on the contrary, `$applyAsync()` always triggers a new cycle).